local-test.sh

```diff
@@ -40,26 +40,29 @@ function restore_workspace_if_requested {
         git checkout .
         git clean -fd
         git stash pop
-        yarn install --frozen-lockfile
+        pnpm install --frozen-lockfile
     else
         echo "Restore not required"
     fi
 }
 
+CURRENT_DIR=${pwd}
```

Comment:
should this be using round brackets?
```suggestion
CURRENT_DIR=$(pwd)
```